src/evaluate.py

```diff
@@ -148,6 +150,9 @@ def write_preds(tasks: Iterable[tasks_module.Task], all_preds, pred_dir, split_n
         elif isinstance(task, CommitmentTask):
```

Comment:
We should really move these into the task objects. Mind making an issue?

Reply:
done in #635